cookbook/session/avoid_session_start.rst

```diff
@@ -0,0 +1,54 @@
+.. index::
+    single: Sessions, cookies
+
+Avoid Starting Sessions for Anonymous Users
+===========================================
+
+Sessions in Symfony applications are automatically started whenever they are necessary.
```

Comment:
```rst
Sessions are automatically started whenever you read, write or even check for the existence
of data in the session. This means that if you need to avoid creating a session cookie for some
users, it can be difficult: you must *completely* avoid accessing the session.

For example, one common problem in this situation involves checking for flash messages, which
are stored in the session. The following code would guarantee that a session is *always* started:
```
... then the code block